scripts/Tools/mkSrcfiles

```diff
@@ -28,13 +28,17 @@ foreach $dir (@paths) {  # (could check that directories exist here)
 # Loop through the directories and add each filename as a hash key.  This
 # automatically eliminates redunancies.
 %src = ();
+my $skip_prefix = $ENV{mkSrcfiles_skip_prefix};
```

Comment:
We need to exclude test_* files when building FMS but some other components need files of the same name, so we add this as an environment variable so it can be applied selectively.